core-web/libs/data-access/src/lib/dot-es-content/dot-es-content.service.ts

```diff
@@ -19,7 +19,7 @@ export interface queryEsParams {
     offset?: string;
     query: string;
     sortField?: string;
-    sortOrder?: ESOrderDirection;
+    sortOrder?: ESOrderDirection | string;
```

Comment:
Why?

Reply:
To pass the query as I explained in the other comment.